samples/games/kotlin/build.gradle.kts

```diff
@@ -115,7 +116,15 @@ kotlin {
             if (this is org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeCompilation) {
                 println("Configuring target ${this.target.name}")
                 this.target.binaries {
-                    sharedLib(listOf(org.jetbrains.kotlin.gradle.plugin.mpp.NativeBuildType.DEBUG))
+                    val libTarget = when(buildTarget?.toLowerCase()) {
+                        "release" -> listOf(org.jetbrains.kotlin.gradle.plugin.mpp.NativeBuildType.RELEASE)
+                        "debug" -> listOf(org.jetbrains.kotlin.gradle.plugin.mpp.NativeBuildType.DEBUG)
+                        else -> {
+                            System.err.println("Build target not specified, defaulting to DEBUG. To set release target, specify: -Ptarget=RELEASE")
```

Comment:
A warning would be sufficient here. Personally i only log errors when they are fatal or non fatal but critical. This is neither of those things and warning is also red 😜